.github/workflows/default.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     uses: ./.github/workflows/00-init.yml
 
   scan-secrets:
+    if: github.event.pull_request == null || github.event.pull_request.head.repo.owner.login == 'db-ui'
```

Comment:
@nmerget I don't get what for you use `github.event.pull_request.head.repo.owner.login == 'db-ui'` here.